input/docs/writing-builds/tasks/criteria.md

```diff
@@ -3,7 +3,9 @@ Description: How to define criterias for tasks
 RedirectFrom: docs/fundamentals/criteria
 ---
 
-You can control and influence the flow of the build script execution by providing criteria. This is a predicate that has to be fulfilled for the task to execute. The criteria does not affect however succeeding task will be executed.
+# Cake script runner
+
+When using [.NET Tool], [Cake runner for .NET Framework] or [Cake runner for .NET Core] you can control and influence the flow of the build script execution by providing criteria. This is a predicate that has to be fulfilled for the task to execute. The criteria does not affect however succeeding task will be executed.
```

Comment:
@pascalberger I know that this isn't a new change, but the wording here is a little unclear to me:

>  The criteria does not affect however succeeding task will be executed.

What exactly is being said here?

Reply:
I'm actually not really happy with the whole page, since it uses quite complex examples might obscure the effective documentation. Also don't understand this sentence. Maybe it is about that criterias don't affect the build graph. I would suggest to merge this PR like it is and rewrite the existing examples afterwards in a separate PR